wptt-webfont-loader.php

```diff
@@ -474,7 +489,7 @@ protected function write_stylesheet() {
 
 			// If we got this far, we need to write the file.
 			// Get the CSS.
-			if ( ! $this->css ) {
+			if ( is_null( $this->css ) ) {
```

Comment:
```suggestion
			if ( null !== $this->css ) {
```

Reply:
Hi @aristath Can you please explain the reason for this change?

Reply:
In WordPress Core we've already replaced all instances of `is_null` with strict `null === $x` checks. 
The reason behind that change was performance (strict checks are faster than an `is_null` call), as well as readability and consistency 👍

Reply:
Thanks for clarification. I've updated the code.

Reply:
Hi there! I noticed that the changes for the stricter `null === $x` check haven't been made yet. I believe this would align better with WordPress Core standards and improve performance. Can we proceed with implementing this check? Thanks!